projects/material-addons/src/themes/common/styles.scss

```diff
@@ -88,11 +104,32 @@ $contrast: (
   } @else {
     @warn 'variable $main-primary  in theme #{$theme-name} is not defined';
   }
+
+  .mat-chip-selected {
+    background-color: white !important;
+  }
+}
+
+@mixin typography($theme-name, $theme) {
+  $typography-config: mat.get-typography-config($theme);
+
+  // set custom typography styles here
```

Comment:
Do we need this comment?

Reply:
I decided to put it here as ist es also a excample how you could extend a theme -> I would keep it